packages/contract-notifier/README.md

```diff
@@ -0,0 +1,29 @@
+# UMA Contracts Notifier
+
+This app continuously attempts to monitor UMA financial contracts and sends notifications based on predefined conditions. Currently the app implements only expiring contracts notifier.
```

Comment:
```suggestion
This app continuously monitors UMA financial contracts and sends notifications based on predefined conditions. Currently the app implements only expiration notifications on LSP contracts.
```

Reply:
Its not limited to LSP, so any expiring contracts served by UMA API would be covered (also EMPs)